aion/web/elm/src/Update.elm

```diff
@@ -121,32 +140,32 @@ update msg model =
             else
                 model ! []
 
-        NoOperation ->
-            model ! []
-
-        SetNewQuestionContent questionContent ->
+        UpdateCreateQuestionForm name value ->
             let
                 oldPanelData =
                     model.panelData
             in
-                { model | panelData = { oldPanelData | newQuestionContent = questionContent } } ! []
+                { model
+                    | panelData =
+                        { oldPanelData | createQuestionForm = Forms.updateFormInput oldPanelData.createQuestionForm name value }
```

Comment:
I think we should leave `Forms` prefix so we know where this function comes from.